src/apprt/gtk/Window.zig

```diff
@@ -582,6 +582,13 @@ pub fn sendToast(self: *Window, title: [:0]const u8) void {
 }
 
 fn displayCrashReportsNoticeIfNeeded(self: *Window, app: *App, parent_widget: *c.GtkWidget) !void {
+    if (!((comptime adwaita.versionAtLeast(1, 3, 0)) and
+        adwaita.enabled(&app.config) and
```

Comment:
As of this morning Adwaita is no longer optional so this function no longer exists.